dev-tools/oxlog/src/lib.rs

```diff
@@ -344,7 +355,7 @@ fn load_svc_logs(dir: Utf8PathBuf, logs: &mut BTreeMap<ServiceName, SvcLogs>) {
             };
 
             logfile.read_metadata(&entry);
-            if logfile.size == Some(0) {
+            if logfile.size == Some(0) && show_empty {
```

Comment:
This condition is very confusing. If `show_empty` is false then we we show the file which is backwards. I believe this stems from the `ArgAction::SetFalse` for the `show_empty` option. While this makes sense on the command line, it doesn't make sense down here in these conditions. What we really want here is `if logfile.size == Some(0) && !show_empty`.

Maybe instead of using `action=ArgAction::SetFalse` for the clap argument, you use `default_value_t = false`. Then you can still pass the parameter down if it is not present, and I think it should also allow you to not have to pass `true` when you set the flag since it's a bool. 

While you could still use `action=ArgAction::SetFalse` and then flip the flag before passing it down, this also seems confusing as now the value the user passed is changing, which I dislike. Anyone who prints the args during debugging may be confused.

Reply:
Okay, I switched it around and now I think it lines up with what you suggested.
The command line behavior is still the same, but the logic and the variable names is
now consistent.